README.md

```diff
@@ -12,7 +14,18 @@ pnpm install
 
 #### Development
 
-Start the development server:
+Be sure to set the following environment variables prior to starting the development server:
+- `COOKIE_SECRET`, should be at least 32 characters long
+- `ADS_SESSION_COOKIE_NAME`
+- `SCIX_SESSION_COOKIE_NAME`
+
+Then you can start the development server with mock API endpoints:
+
+```bash
+pnpm dev:mocks
+```
+
+Or, run the development server against live APIs:
 
 ```bash
 pnpm dev
```

Comment:
It looks like this requires an API proxy? Is there a command to run before this to ensure the dev frontend can access the backend?